src/main/java/com/mx/iwaconsolti/calculator/Divicion.java

```diff
@@ -6,7 +6,7 @@
 public class Divicion {
```

Comment:
Since you are creating this bean manually in the configuration, it doesnt require the component annotation